frontend/src/features/modalSlice.ts

```diff
@@ -0,0 +1,35 @@
+import { createSlice, PayloadAction } from '@reduxjs/toolkit';
```

Comment:
Haven't gone through this too finely yet, but just been going through the current PRs because I found (what I think at least) is an error in how we're using slices (if we're using duck folder structure).

I'm pretty sure that a slice is meant to be in the same component file that it'll be used with.
So this should go in the `employees` directory. Or the `rootModal` one, not really sure tbh.

That way we'd have related stuff in one folder.

Reply:
Okay, so this I guess depends on the exact folder structure we use. In my grad project we had components separate from redux store stuff. Maybe this calls fro more discussion